### PR TITLE
Release docs

### PR DESCRIPTION
**Pull request by jaredhirsch**
@chuckharmston R?

Bump version to 1.0.1 and make the release docs more verbose.

Nicely formatted release docs: https://github.com/6a68/universal-search/blob/release-docs/docs/building.md

### CODE REVIEW

File: docs/building.md [hunk 1]
```diff
@@ -95,11 +95,13 @@ Updated: https://s3-us-west-2.amazonaws.com/universal-search/universal-search.xp
...
+1. Increment the version numbers in `update.rdf` and `install.rdf`. The new version number must be higher than the previous versions.
+2. `git commit` the rdf file changes, with a commit message of the form "Version a.b.c".
+3. Run `make`.
+4. Verify that the add-on was correctly uploaded by downloading the URL demonstrated by the output, e.g.
+
+  ```shell
+  $ curl -I https://s3-us-west-2.amazonaws.com/universal-search/universal-search-1.0.0.xpi
+  HTTP/1.1 200 OK
+  ```
+5. Push the new version commit, and the newly-created git tag, to the main Mozilla repo.
```
Comment (chuckharmston) [inline docs/building.md]: Maybe omit the word `version` here? It reads a little awkwardly.

**Comment by chuckharmston** [comment]
r+, one minor suggestion. Merge at will!

File: docs/building.md (hunk 1 above)
Comment (jaredhirsch) [inline docs/building.md]: Sounds good

**Comment by jaredhirsch** [comment]
Thanks! I'll fix and merge

**Pull request merged**